VideoWeb/VideoWeb/appsettings.Development.json

```diff
@@ -1,9 +1,9 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Debug",
-      "System": "Debug",
-      "Microsoft": "Debug",
+      "Default": "Warning",
+      "System": "Warning",
+      "Microsoft": "Warning",
```

Comment:
Don't thing we need to commit changes to the development.json

Reply:
that's alright. I thought we were going to lower the logs for dev too after the feedback from the las retro